[2024-05-01T10:21:49.946+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-01T10:21:49.961+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final_exam.insert_db manual__2024-05-01T10:21:05.315844+00:00 [queued]>
[2024-05-01T10:21:49.967+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final_exam.insert_db manual__2024-05-01T10:21:05.315844+00:00 [queued]>
[2024-05-01T10:21:49.967+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-01T10:21:49.978+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): insert_db> on 2024-05-01 10:21:05.315844+00:00
[2024-05-01T10:21:49.983+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=114) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-01T10:21:49.984+0000] {standard_task_runner.py:63} INFO - Started process 116 to run task
[2024-05-01T10:21:49.985+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final_exam', 'insert_db', 'manual__2024-05-01T10:21:05.315844+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/Project.py', '--cfg-path', '/tmp/tmpxcufn915']
[2024-05-01T10:21:49.986+0000] {standard_task_runner.py:91} INFO - Job 35: Subtask insert_db
[2024-05-01T10:21:50.018+0000] {task_command.py:426} INFO - Running <TaskInstance: Final_exam.insert_db manual__2024-05-01T10:21:05.315844+00:00 [running]> on host 73ada3aeb297
[2024-05-01T10:21:50.080+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final_exam' AIRFLOW_CTX_TASK_ID='insert_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-01T10:21:05.315844+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-01T10:21:05.315844+00:00'
[2024-05-01T10:21:50.081+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-01T10:22:43.521+0000] {logging_mixin.py:188} INFO - Successfully.
[2024-05-01T10:22:43.536+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-01T10:22:43.537+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-01T10:22:43.544+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Final_exam, task_id=insert_db, execution_date=20240501T102105, start_date=20240501T102149, end_date=20240501T102243
[2024-05-01T10:22:43.582+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-01T10:22:43.599+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-01T10:22:43.602+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
